src/collection/forEachRight/index.js

```diff
@@ -0,0 +1,20 @@
+const { isObject } = require('../../lang/isObject/index.js')
+const { isArray } = require('../../lang/isArray/index.js')
+
+function forEachRight (collection, cb) {
+  if (isArray(collection)) {
+    collection.reverse()
```

Comment:
This will change the input array. That is wrong. Just duplicate the input array and reverse that new one.